install/kubernetes/helm/istio/charts/security/templates/deployment.yaml

```diff
@@ -40,6 +40,9 @@ spec:
             - --root-cert=/etc/cacerts/root-cert.pem
             - --cert-chain=/etc/cacerts/cert-chain.pem
           {{- end }}
+          {{- if .Values.identityDomain }}
```

Comment:
you may also want to update the istio-remote helm chart for security.

Reply:
true. Will handle that in a separate PR. Based on feedback, the value needs to be sync'ed across multiple components.